rs/ic_os/network/src/lib.rs

```diff
@@ -13,6 +14,35 @@ pub mod info;
 pub mod interfaces;
 pub mod systemd;
 
+/// Write SetupOS or HostOS systemd network configuration.
+/// Requires superuser permissions to run `ipmitool` and write to the systemd directory
+/// TODO(NODE-1466): Consolidate generate_network_config_new_config and generate_network_config
+pub fn generate_network_config_new_config(
+    network_settings: &NetworkSettings,
+    generated_mac: &MacAddr6,
+    output_directory: &Path,
+) -> Result<()> {
+    eprintln!("Generating IPv6 address");
```

Comment:
When do we log to stderr and when do we log to stdout? E.g. above we log to stdout.

Reply:
Honestly, I don't know. Perhaps we should ticket this because I know we use one vs the other in multiple places and sometimes both in the same script for no discernible reason.